bluebrain/repo-bluebrain/packages/nmodl/package.py

```diff
@@ -47,7 +47,7 @@ class Nmodl(CMakePackage):
     depends_on('python@3.6.0:')
     depends_on('py-jinja2@2.10:')
     depends_on('py-pytest@4.0.0:')
-    depends_on('py-sympy@1.3:')
+    depends_on('py-sympy@1.3:1.8')
```

Comment:
@olupton happier?

Reply:
I think I can solve the problem without locking sympy to the most recent version

Reply:
Yep, all good. I was confident it would know about the version, less sure if it would be re-installed or not.